benchmarks/infill_density/infill_ascii.prm

```diff
@@ -0,0 +1,199 @@
+# mu = elastic shear modulus
+# tm is the time scale of viscous stress relaxation for the mantle.
+# This allows for virtually all stresses in the mantle to be relaxed
+# and for the elastic lithosphere to be fully compensating the load.
+# The analytic solution for this setup is outlined in the Generic
+# Mapping Tools function grdflexure:
+# https://docs.generic-mapping-tools.org/6.1/supplements/potential/grdflexure.html
+
+#  Global parameters
+set Additional shared libraries            = ./libinfill_density.so
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 120e3
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, single Stokes
+set CFL number                             = 0.1
+set Maximum time step                      = 750
+set Output directory                       = output
+set Pressure normalization                 = no
+
+# Model geometry (500x500 km, 50 km spacing)
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 15
+    set Y repetitions = 30
+    set X extent      = 1200e3
+    set Y extent      = 1200e3
+  end
+end
+
+# Mesh refinement specifications
+# Refine the mesh in the uppermost part of the model
+# to better capture lithospheric response to the load.
+subsection Mesh refinement
+  set Initial adaptive refinement              = 2
+  set Initial global refinement                = 1
+  set Time steps between mesh refinement       = 1
+  set Minimum refinement level                 = 0
+  set Strategy                                 = minimum refinement function, strain rate
+  subsection Minimum refinement function
+    set Coordinate system                      = cartesian
+    set Variable names                         = x,y,t
+    set Function expression                    = if(y>=1050e3, 3, 0)
+  end
+end
+
+# Element types
+subsection Discretization
+  set Composition polynomial degree           = 2
+  set Stokes velocity polynomial degree       = 2
+  set Temperature polynomial degree           = 1
+  set Use locally conservative discretization = true
+  set Use discontinuous temperature discretization = false
+  set Use discontinuous composition discretization = true
+  subsection Stabilization parameters
+      set Use limiter for discontinuous composition solution = true
+      set Global composition maximum =  1.e13,  1.e13,  1.e13, 1.0
+      set Global composition minimum = -1.e13, -1.e13, -1.e13, 0.0
+  end
+end
+# Formulation classification
+subsection Formulation
+  set Enable elasticity = true
+end
+
+subsection Mesh deformation
+  set Mesh deformation boundary indicators = top: free surface
+  set Additional tangential mesh velocity boundary indicators = left, right
+  subsection Free surface
+    set Surface velocity projection = vertical
+  end
+end
+
+# Velocity boundary conditions
```

Comment:
Can you add a comment here (or in the section below) about why the left boundary is free-slip and the right is open? I assume this helps remove artifacts from return flow induced by the surface loading?